feature/.template/install.sh

```diff
@@ -0,0 +1,27 @@
+# The installer script for your feature, which is called in the init phase of CloudControl
+
+# Include the feature installer utils which include helpers for the installation of your feature
+# See /assets/feature-installer-utils.sh
+
+. /feature-installer-utils.sh
+
+# Create a temporary directory to work in
+prepare
```

Comment:
See /feature-installer-utils.sh
If named different, you wouldn't need the comment